build-yaneuraou.sh

```diff
@@ -14,16 +14,27 @@ fi
 echo "- Determining CPU architecture..."
 
 ARCH=SSE42
+COMP=clang++
```

Comment:
Not sure about this line, but in the [source Makefile](https://github.com/yaneurao/YaneuraOu/blob/master/source/Makefile#L69) it defaults to `clang++` which is a bit confusing because of the produced filename with `gcc` in the end.😑